pkgs/by-name/ba/bash-env-json/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  stdenvNoCC,
+  lib,
+  makeWrapper,
+  fetchFromGitHub,
+  coreutils,
+  gnused,
+  jq,
+  nix-update-script,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "bash-env-json";
+  version = "0.9.1";
+
+  src = fetchFromGitHub {
+    owner = "tesujimath";
+    repo = "bash-env-json";
+    rev = finalAttrs.version;
+    hash = "sha256-cZEkYOr9z6yLPA4PSo6+hogaqb1vhWaYi/rp4asfsbM=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    cp bash-env-json $out/bin
+
+    runHook postInstall
+  '';
+
+  postFixup = ''
+    wrapProgram $out/bin/bash-env-json --set PATH ${
```

Comment:
```suggestion
    wrapProgram $out/bin/bash-env-json --prefix PATH : ${
```

Reply:
is there another reason like with full paths why prefix is actually better despite things like https://gist.github.com/CMCDragonkai/9b65cbb1989913555c203f4fa9c23374 saying that `one should always try to use --set instead of prefix`?

Reply:
> One should always try to use --set instead of --prefix because you shouldn't rely on the user profile environment variables.

is a good point, but in fact, many people don't write this list in full, or make a mistake in upgrade, and in some cases (that I don’t know about) it will call the executable specified by the parameter. So overall I don't want to have this kind of trouble.

Reply:
Ok, done